tests/virt/node/migration_and_maintenance/test_post_copy_migration.py

```diff
@@ -142,6 +143,10 @@ def test_hotplug_cpu(self, hotplugged_sockets_memory_guest, hotplugged_vm, vm_ba
     )
     @pytest.mark.dependency(depends=[f"{TESTS_CLASS_NAME}::hotplug_cpu"])
     @pytest.mark.polarion("CNV-11424")
-    def test_hotplug_memory(self, hotplugged_sockets_memory_guest, hotplugged_vm, vm_background_process_id):
+    def test_hotplug_memory(self, request, hotplugged_sockets_memory_guest, hotplugged_vm, vm_background_process_id):
+        # Dynamically skip only this test when class is run with s390x params as memory hotplug is not supported
+        if request.node.get_closest_marker("s390x"):
+            pytest.skip("Skipping test_hotplug_memory for s390x")
```

Comment:
please move this to separate fixture `skip_on_s390x_architecture` and add with `@pytest.mark.usefixtures("skip_on_s390x_architecture")`

Reply:
please mark all `s390x` applicable tests with a marker.
we should not have such a skip in the code